engines/dfc_provider/spec/requests/enterprise_groups/affiliated_by_spec.rb

```diff
@@ -70,4 +70,36 @@
       end
     end
   end
+
+  path "/api/dfc/enterprise_groups/{enterprise_group_id}/affiliated_by/{id}" do
+    delete "Remove enterprise from group" do
+      parameter name: :enterprise_group_id, in: :path, type: :string
+      parameter name: :id, in: :path, type: :string
+
+      let(:enterprise_group_id) { group.id }
+      let(:id) { enterprise2.id }
+
+      response "204", "no content" do
+        before do
+          group.enterprises << enterprise2
+          group.save!
```

Comment:
I don't think saving is needed.

Reply:
I would have re-declared `let(:group)` to just create it with both enterprises in the first place, whihc would be more efficient. But I guess it barely makes a difference..